src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenOnBehalfOfParameterBuilder.cs

```diff
@@ -161,6 +161,20 @@ public AcquireTokenOnBehalfOfParameterBuilder WithCcsRoutingHint(string userName
 
             this.WithExtraHttpHeaders(ccsRoutingHeader);
             return this;
+        }
+
+        /// <summary>
```

Comment:
The comments for these new apis should be updated to clearly indicate how the apis should be used. Basically warn against the api for anything other than backwards compatability. ALso, the wiki should be updated to warn against using `InitiateLongRunningProcessInWebApi` over and over again.